understanding/20/change-on-request.html

```diff
@@ -56,8 +56,7 @@ <h2>Benefits of Change on Request</h2>
          
          <li>
             
-            <p>Individuals who are unable to detect changes of context or may not realize that the
-               context has changed are less likely to become disoriented while navigating a site.
+            <p>Individuals who may not realize that the context has changed if they did not initiate the change themselves are less likely to become disoriented while navigating a site.
```

Comment:
The original issue was triggered because the benefit statement was interpretted as a problem statement. By adding the 'solution' to the benefit statement, it becomes clear it is in the form of a [problem: solution] phrase
 ```suggestion
            <p>Individuals who are unable to detect changes of context or who may not realize that the context has changed are less likely to become disoriented if they initiate the change themselves.
```

Reply:
either one works for me (though i still think the first part says the same thing twice... "are unable to detect changes of context" OR "may not realize the context has changed" seems the same, with perhaps subtle shades of severity only)